nuxt.config.ts

```diff
@@ -90,9 +91,20 @@ export default defineNuxtConfig({
 		'~/assets/css/main.css',
 	],
 	modules: [
+		'@primevue/nuxt-module',
 		'@pinia/nuxt',
 	],
 	devtools: { enabled: true },
+	primevue: {
+		options: {
+			theme: {
+				preset: Aura,
+			},
+		},
+		components: {
+			prefix: 'pv',
+		},
+	},
 	vite: {
```

Comment:
`time nuxt build`

before this PR:
```
real    0m21.259s
user    0m0.092s
sys     0m0.214s
```

after this PR (sometimes it's half that, but that's still too much):
```
real    3m10.533s
user    0m0.140s
sys     0m0.355s
```